src/Model/Lumberjack.php

```diff
@@ -155,7 +178,7 @@ protected function shouldFilter()
     {
         $controller = Controller::curr();
         return $controller instanceof LeftAndMain
-            && in_array($controller->getAction(), array('treeview', 'listview', 'getsubtree'));
+            && in_array($controller->getAction(), array('show', 'index', 'treeview', 'listview', 'getsubtree'));
```

Comment:
Also, using such a long array inline is a bit unreadable, it would be better to extract it to a variable above the return.